docs/self-managed/operate-deployment/data-retention.md

```diff
@@ -33,6 +33,20 @@ In case of intensive Zeebe usage, the amount of data can grow significantly over
 
 Dated indices may be safely removed from Elasticsearch. "Safely" means only finished process instances are deleted together with all related data, and the rest of the data stays consistent. You can use Elasticsearch Curator or other tools/scripts to delete old data.
 
+Users updating from Elasticsearch 7 to Elasticsearch 8, will encounter issues with the Elasticsearch Curator. To resolve this, Operate allows configuring an Index Lifecycle Management (ILM) Policy using the `archiver` configuration options:
+
+### A snippet from application.yml
+
+```yaml
+camunda.operate:
+  archiver:
+    waitPeriodBeforeArchiving: 5m
+    ilmEnabled: true
+    ilmMinAgeForDeleteArchivedIndices: 10m
```

Comment:
I think we actually need to change this!
These values were proposed for testing, but customers should use the following in my opinion:
```
camunda.operate:
  archiver:
    ilmEnabled: true
    ilmMinAgeForDeleteArchivedIndices: 30d
```

Reply:
Actually should we also mention, that `ilmMinAgeForDeleteArchivedIndices` defines amount of time to store archived data before deletion. And the values use Elasticsearch TimeUnit format: https://www.elastic.co/guide/en/elasticsearch/reference/current/api-conventions.html#time-units

Reply:
I made the changes.